src/components/big-interactive-pages/editor.tsx

```diff
@@ -151,6 +152,47 @@ export const saveGame = debounce(
 	}
 );
 
+export async function saveGame2(persistenceState: Signal<PersistenceState>) {
```

Comment:
suggest better name of this method implying how it functions in a uniqueish way - 2 is noninformational, this "starts the saving process for a game" or something like that

Reply:
solved